core/src/main/scala/cats/syntax/option.scala

```diff
@@ -0,0 +1,20 @@
+package cats
+package syntax
+
+import cats.data.Xor
+
+trait OptionSyntax {
+  def none[A] = Option.empty[A]
+  implicit def toOptionSyntax[A](a: A): ToOptionOps[A] = new ToOptionOps(a)
+  implicit def optionSyntax[A](a: Option[A]): OptionOps[A] = new OptionOps(a)
+}
+
+class ToOptionOps[A](val a: A) extends AnyVal {
+  def some: Option[A] = Option(a)
+}
+
+class OptionOps[A](val a: Option[A]) extends AnyVal {
+  def toLeftXor[B](b: => B): A Xor B = a.fold[A Xor B](Xor.Right(b))(Xor.Left(_))
+  def toRightXor[B](b: => B): B Xor A = a.fold[B Xor A](Xor.Left(b))(Xor.Right(_))
```

Comment:
It can be nice in a context where you have several things in a comprehension that you want to lift. 

```scala
for {
  a <- foo(...) \/> "oops foo"
  b <- bar(a)   \/> "oops bar"
} yield whatevs
```

scalaz is full of little things like this and I like them, but as you say it's not a big deal. You could argue that this is the kind of thing that scares people away.